website/source/docs/guides/outage.html.md

```diff
@@ -82,14 +105,18 @@ servers were lost, so information about what's committed could be incomplete.
 The recovery process implicitly commits all outstanding Raft log entries, so
 it's also possible to commit data that was uncommitted before the failure.
 
-See the [section below](#peers.json) for details of the recovery procedure. You
+See the section below on manual recovery using peers.json for details of the recovery procedure. You
```

Comment:
No link?

Reply:
It seems likely that the next section will actually be on the same page, so the link might not be necessary. We've also been inconsistent with linking, which is a topic we should revisit next year.